### PR TITLE
add and edit new html pages and css files

### PR DESCRIPTION
**Pull request by liebeckm**

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -3,4 +3,6 @@
   background-color: #29282a;
   border: 0.1rem solid #e6664e;
   padding: 1rem;
+  width: 25%;
 }
+/*not sure how to position the button in the center of the card, any ideas?*/
```
Comment (alexWaligorski) [inline components/button.css]: put a <div> or <span> element around the button.

File: components/footer.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+footer {
+  position: fixed;
+  bottom: 0rem;
+  width: 100%;
+  background-color: #be7f6a;
+}
+
+/*there is this gap on the left side of the footer, any ideas how I could remove?*/
```
Comment (alexWaligorski) [inline components/footer.css]: did you set the margin of your body to 0 ? this could help.

File: components/profile-pic.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.profile-pic {
+  width: 400;
+  height: auto;
+}
+
+/* I have no idea why this class does not work here, instead I added width in profile.html image element directly */
```
Comment (alexWaligorski) [inline components/profile-pic.css]: you need to specify pixel or % or em..

File: components/profile-pic.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.profile-pic {
+  width: 400;
```
Comment (alexWaligorski) [inline components/profile-pic.css]: pixel?

File: pages/bookmark.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            <a href="./profile.html" class="navbar--element">PROFILE</a>
+          </li>
+        </ul>
+      </nav>
+    </footer>
+  </body>
+</html>
+
+<!--TO DO: Make sure that only bookmarked questions are on this page.
+Update the navbar so that the bookmark icon is highlighted.-->
```
Comment (alexWaligorski) [inline pages/bookmark.html]: you don't need javascript fot that. only change the background-color of the nav-item that is corresponding with the page it is on.

**Review by alexWaligorski — COMMENTED** [review]
very nicely structured project folders and css! only some small remarks.

File: components/button.css (hunk 1 above)
Comment (liebeckm) [reply components/button.css]: solved it with `align items: center;` 🥳

File: components/footer.css (hunk 2 above)
Comment (liebeckm) [reply components/footer.css]: added `margin: -0.65rem` but this is probably not the cleanest solution.

**Pull request merged**